nni/retiarii/integration.py

```diff
@@ -121,7 +122,19 @@ def send_trial(self, parameters, placement_constraint=None):
             'placement_constraint': placement_constraint
         }
         _logger.debug('New trial sent: %s', new_trial)
-        send(CommandType.NewTrialJob, nni.dump(new_trial))
+
+        send_payload = nni.dump(new_trial, pickle_size_limit=-1)
+        if len(send_payload) > 50 * 1024:
```

Comment:
50KB is too small? how about 1MB or 10MB?

Reply:
I'd rather play safe. In most of my tests, the whole payload is actually less than 2KB.
Judging from the pipe limit is 1MB (maybe my memory goes wrong), Maybe 256KB is still acceptable?